controllers:
  joint_state_controller:
    type: joint_state_controller/JointStateController
    publish_rate: 50
  windmill_controller:
    publish_rate: 50
    type: windmill_controller/WindMillController
    pid_control:
      p: 200.0
      i: 0.0
      d: 0.6
      i_max: 0.0
      i_min: 0.0
      Target: 0.0
windmill_mode: 1

 # left_front_wheel_joint_controller:
  #  type: effort_controllers/JointPositionController
  #  joint: left_front_wheel_joint
 #   pid: {p: 100.0,i: 0.01,d: 10.0}
 # left_back_wheel_joint_controller:
 #   type: effort_controllers/JointPositionController
 #   joint: left_back_wheel_joint
 #   pid: {p: 100.0,i: 0.01,d: 10.0}
 # right_front_wheel_joint_controller:
  #  type: effort_controllers/JointPositionController
  #  joint: right_front_wheel_joint
 #   pid: {p: 100.0,i: 0.01,d: 10.0}
 # right_back_wheel_joint_controller:
 #   type: effort_controllers/JointPositionController
 #   joint: right_back_wheel_joint
 #   pid: {p: 100.0,i: 0.01,d: 10.0}
#  diff_drive_controller:
#    type: "diff_drive_controller/DiffDriveController"
 #   left_wheel: ['left_front_wheel_joint','left_back_wheel_joint']
#    right_wheel: ['right_front_wheel_joint','right_back_wheel_joint']
 #   publish_rate: 50
#    pose_covariance_diagonal: [0.001, 0.001, 1000000.0, 1000000.0, 1000000.0, 0.03]
#    twist_covariance_diagonal: [0.001, 0.001, 0.001, 1000000.0, 1000000.0, 0.03]
